### PR TITLE
text and test updated for income summary page

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -499,7 +498,7 @@ tai.income.calculation.updateLink.regular=update your estimated taxable income
 tai.income.details.addMissingCompanyCar=Add a missing company car
 tai.income.details.benefit.amount.announce=- The amount of this benefit is {0}.
 tai.income.details.benefit.name.announce=Benefit type:
-tai.income.details.checkPaymentsReceived.screenReader=Check payments received from {0}.
+tai.income.details.checkPaymentsReceived.with.employer=Check payments received from {0}
```
Comment (struong) [inline conf/messages:501]: Is removing the full stop intentional? As you have kept it in `tai.income.details.yourEstimatedTaxableIncome.with.amount`

File: app/views/IncomeSourceSummary.scala.html [hunk 2]
```diff
@@ -84,32 +84,18 @@
 
     @if(model.rtiAvailable) {
         @if(model.isPension) {
```
Comment (struong) [inline app/views/IncomeSourceSummary.scala.html:86]: @anujakhare @vimalsuresh when it comes to testing this ticket you will need to test a pension user and an employment user

File: conf/messages (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply conf/messages:501]: yeah as its a link

Comment (Muhammad-Ali-Ahmad) [reply conf/messages:501]: image

**Review by struong — APPROVED** [review]

**Pull request merged**